azuredeploy.tf

```diff
@@ -78,9 +78,9 @@ resource "random_string" "secret" {
 resource "azuread_application" "test" {
   count                      = data.azurerm_client_config.current.service_principal_application_id == "" ? 1 : 0
   name                       = "eventhubstest"
-  homepage                   = "https://eventhubstest"
-  identifier_uris            = ["https://eventhubstest"]
-  reply_urls                 = ["https://eventhubstest"]
+  homepage                   = "https://eventhubstest-${random_string.name.result}"
```

Comment:
Without this random seed, the terraform setup was complaining about the identifierUri having already been used. Since these fields are just to make the AAD app happy, adding the random piece avoids the conflict without changing the behavior.